docs/user-guides/zli.md

```diff
@@ -300,3 +300,36 @@ This command lists all repositories in the zot registry.
       -u, --user string   User Credentials of zot server in "username:password" format
 
     Run 'zli config -h' for details on [config-name] argument
+
+### zli search
+The `search` command has been added and it allows smart searching for repos by name or images by their repo:tag
```

Comment:
Edit: "The `search` command allows smart searching for repositories by name or for images by their repo:tag"